packages/flutter/test/tracing/README.md

```diff
@@ -0,0 +1,7 @@
+# Tracing tests
```

Comment:
Should this be removed?

Reply:
The whole file or this line?

I think most of the folders in `dev/` have a README.md - the goal of this one is to make it clear what kind of tests belong in here, and what's special about running them.

Reply:
This isn't in dev though, this is in `flutter/test`

Reply:
Ah! Thanks, good catch.  Removed.